### ISSUE TITLE
`free-layout-zoom` fails

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3240]
`free-layout-zoom` fails with this traceback:
```
Traceback (most recent call last):

  File "c:\Tom\git\leo-editor\leo\core\leoKeys.py", line 2507, in callAltXFunction
    func(event)

  File "c:\Tom\git\leo-editor\leo\plugins\free_layout.py", line 492, in free_layout_zoom
    c.free_layout.get_top_splitter().zoom_toggle()

  File "c:\Tom\git\leo-editor\leo\plugins\nested_splitter.py", line 1123, in zoom_toggle
    focused = Qt.QApplication.focusWidget()

AttributeError: type object 'Qt' has no attribute 'QApplication'
```

**Comment by edreamleo** [comment]
@tbpassin Thanks for this report. I'm on it.

**Comment by edreamleo** [comment]
Closed via PR #3242.
